boards/st/stm32l562e_dk/stm32l562e_dk_stm32l562xx_ns.dts

```diff
@@ -41,24 +41,64 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 
+		/*
+		 * Flash layout:
+		 *  - BL2 - Multi image boot
+		 *  - internal
+		 *  - external
+		 *  - Scratch
+		 */
 		boot_partition: partition@0 {
-			reg = <0x00000000 DT_SIZE_K(100)>;
+			reg = <0x00000000 DT_SIZE_K(68)>;
 			read-only;
 		};
 
-		/* Secure image primary slot */
+		scratch_partition: partition@11000 {
+			reg = <0x00011000 DT_SIZE_K(8)>;
+		};
```

Comment:
not a problem with the PR but just to point out that assuming the 70% of the application space here is used, that would be about ~330KiB and one single firmware update would do 41 or 82 (can't remember if it uses it when writing in one or both directions) flash erase/write cycles on this section here, meaning the flash endurance cycles will go down far faster than the rest of the device (for a 10k endurance flash device that means ~123 or ~246 firmware updates)

Reply:
I complete agreed with you here and I'll propose a patch to mainline.